debian/wazo-asterisk-config.postinst

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+# postinst script
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+    configure)
+        if [ -d /run/systemd/system ]; then
```

Comment:
This check is redundant with the `|| true`
we should remove it, or remove `|| true`